src/auth/Auth.sol

```diff
@@ -13,6 +13,8 @@ abstract contract Auth {
 
     Authority public authority;
 
+    error UNAUTHORIZED();
```

Comment:
these should all be camel case

Reply:
Word, just kept the `"UNAUTHORIZED"` revert message.